meta.ts

```diff
@@ -274,6 +274,12 @@ function parseDocComments(code: string, modulePath: string) {
   } as Builder.ParsedModule;
 }
 
+/**
+ * Keys available for binding as the system clipboard register.
+ * These are available in addition to the default `"` and `null`
+ */
+export const availableClipboardRegisters = [..."abcdefghijklmnopqrstuvwxyz".split(""), "*", "+"];
```

Comment:
```suggestion
export const availableClipboardRegisters = [..."abcdefghijklmnopqrstuvwxyz", "*", "+"];
```

Reply:
Also, it might be relevant to remove a few letters from the alphabet, as some letters are mapped to special registers (e.g. **d**quote is also mapped to `d` and `D`).

Reply:
> Also, it might be relevant to remove a few letters from the alphabet, as some letters are mapped to special registers (e.g. dquote is also mapped to d and D).
Is this true? I don't see any evidence of this in Dance or Kakoune.

Reply:
Okay I see where this is happening in Dance, but it doesn't seem quite right as half the bindings overwrite the other half, e.g `D` and `d` don't map to `dquote` but `dot`.

I'll remove `acdhspu` from the available registers for the system clipboard but you might want to add explicit definitions for the alternate bindings of special registers.

Reply:
Welp, that's a great point. I don't remember why it was made this way, but there was definitely an oversight on my part. This is a different issue, so thanks for changing this in the meantime!